pybars/_compiler.py

```diff
@@ -595,9 +595,9 @@ def add_block(self, symbol, arguments, nested, alt_nested):
                 u"    options['inverse'] = lambda this: None\n"
                 ])
         self._result.grow([
-            u"    value = helper = helpers.get(u'%s')\n" % symbol,
+            u"    value = helper = helpers.get(%s)\n" % repr(symbol),
```

Comment:
No tests fail when this is reverted

Reply:
@wbond I tried a lot of test-cases to break the following code but failed.

`helpers.get(u'%s')\n" % symbol`

Any symbol I put over here (i.e. **block helper name**) was breaking the template e.g. #foo' #foo; #foo.foo.

Reply:
So I think this specific change doesn't add any value. Unless I missed something.